dev/multipart-unit-test-generator.js

```diff
@@ -0,0 +1,18 @@
+const http = require("http");
```

Comment:
Hi @akainth015 - looks like this is not really required by the unit test?  Is it only used for running manual tests?

Reply:
Not for manual tests, it generates `tests/multipart` for the `MultiImageInput` test, and I have put it into Git so theoretically we never need to run it again. I have included it just in case something happens and we end up needing to regenerate the file.

Reply:
if it's for automated unit tests, ideally this file should be generated from your python test code and only saved to a temporary directory